src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotification.java

```
@@ -247,11 +249,14 @@ public void run() {
                                     // status. JobCheckOutListener is now responsible for setting the pending status.
                                     return;
                                 }
-                                GitHubBuildStatusNotificationStrategy strat = sourceContext.notificationStrategy();
-                                GitHubBuildStatusNotificationBundle bundle = strat.createNotification(job, null,
-                                        null, source, head);
-                                repo.createCommitStatus(hash, bundle.getState(), bundle.getUrl(), bundle.getMessage(),
-                                        bundle.getContext());
+                                AbstractGitHubNotificationStrategy strat = sourceContext.notificationStrategy();
+                                GitHubNotificationContext notificationContext = new GitHubNotificationContext(job, null,
+                                        source, head);
+                                List<GitHubNotificationRequest> details = strat.notifications(notificationContext, null);
+                                for (GitHubNotificationRequest request : details) {
+                                    repo.createCommitStatus(hash, request.getState(), request.getUrl(), request.getMessage(),
```

we should be using the ignore error here too